backend/src/main/kotlin/com/tngtech/apicenter/backend/connector/rest/controller/ServiceController.kt

```diff
@@ -54,7 +54,7 @@ class ServiceController @Autowired constructor(
     private fun getConsistentId(specificationFileDto: SpecificationFileDto, specificationIdFromPath: String): String {
         specificationFileDto.id?.let {
             idFromFile -> if (idFromFile != specificationIdFromPath)
-                throw MismatchedServiceIdException(idFromFile, specificationIdFromPath)
+                throw MismatchedServiceIdException(listOf(idFromFile, specificationIdFromPath))
```

Comment:
Why not a `Pair`? Then it would be clearer that the two `String`s are related to each other.